schema_utils_test.py

```diff
@@ -581,3 +584,304 @@ def test_notification_email_list_validator(self):
         invalid_vals = [[u'admin@oppia'], big_email_list,
                         [u'admin@oppia.commmm'], [u'a@.com']]
         self.check_normalization(schema, mappings, invalid_vals)
+
+
+class CustomizationArgsUtilUnitTests(test_utils.GenericTestBase):
+    """Test customization args generation and validation."""
+
+    def test_get_full_customization_args(self):
+        """Test get full customization args method."""
+        ca_specs1 = interaction_registry.Registry.get_interaction_by_id(
+            'Continue').customization_arg_specs
+
+        customization_args1a = {
+            'buttonText': {'value': 'Please Continue'}
+        }
+
+        customization_args1b = {
```

Comment:
Rename to: `customization_args_with_extra_arg`

Reply:
Done.